tv_apps/smart-home/style/home.css

```diff
@@ -50,6 +50,7 @@ section {
   height: 33.6rem;
   top: 32.41%;
   overflow: visible;
+  transition: top 0.4s;
```

Comment:
May you use `transform` with `translateY` instead? That performs better.
refer:
http://www.paulirish.com/2012/why-moving-elements-with-translate-is-better-than-posabs-topleft/